railgun/internal/proxy/clientgo_cached_proxy_resolver.go

```diff
@@ -218,7 +211,9 @@ func (p *clientgoCachedProxyResolver) updateKongAdmin() error {
 	// apply the configuration update in Kong
 	timedCtx, cancel := context.WithTimeout(p.ctx, 10*time.Second)
 	defer cancel()
-	_, err = sendconfig.PerformUpdate(timedCtx, logrus.StandardLogger(), &p.kongConfig, true, false, targetConfig, nil, nil, nil)
+
+	// use the previous SHA to determine whether or not to perform an update
```

Comment:
Line 43 in `PerformUpdate` would probably be a better place for this comment.

This is because this function is not concnerned with the logic described by this comment.

Reply:
:+1: 8db2228638a070f1403caeab7fa8304504f08f7d